layouts/partials/footer.html

```diff
@@ -1,5 +1,5 @@
 <footer class="site-footer">
-	<p class="text">&copy; {{ now.Format "2006" }} - {{ .Site.Params.copyright | markdownify }}</p>
+	<p class="text">&copy; {{ now.Format "2006" }} - {{ .Site.Params.copyright | default "Released under the MIT license<br>Powered by [Hugo](//gohugo.io/) with the [Type Theme](//github.com/digitalcraftsman/hugo-type-theme)" | markdownify }}</p>
```

Comment:
URL mean go to HTTPS if you're looking at the page trough HTTPS, and go to HTTP otherwise. It's incorrect place for that (here one should just send people to HTTPS without any choice), but I've just set it to the same value it's set in default config.